ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -447,12 +447,13 @@ export default class SecurityTab extends PureComponent {
           <span>{t('useCollectibleDetection')}</span>
           <div className="settings-page__content-description">
             {t('useCollectibleDetectionDescription')}
-            <br />
-            {t('useCollectibleDetectionDescriptionLine2')}
             <ul className="settings-page__content-unordered-list">
+              <li>{t('useCollectibleDetectionDescriptionLine2')}</li>
               <li>{t('useCollectibleDetectionDescriptionLine3')}</li>
               <li>{t('useCollectibleDetectionDescriptionLine4')}</li>
             </ul>
+            <br />
```

Comment:
Rather than using `<br />`, could we use `<Typography>` around `{t('useCollectibleDetectionDescriptionLine5')}` instead, adding `paddingTop={3}` or something to that effect?

Reply:
`Text` is the new component we have in the `component-library` as a replacement for Typography. I think we should use that

Reply:
Ended up using the Text component, thanks for the suggestions